pandas/io/feather_format.py

```diff
@@ -123,9 +124,16 @@ def read_feather(
         path, "rb", storage_options=storage_options, is_text=False
     ) as handles:
         if dtype_backend is lib.no_default and not using_pyarrow_string_dtype():
-            return feather.read_feather(
-                handles.handle, columns=columns, use_threads=bool(use_threads)
-            )
+            with warnings.catch_warnings():
+                warnings.filterwarnings(
+                    "ignore",
+                    "make_block is deprecated",
+                    DeprecationWarning,
+                )
+
+                return feather.read_feather(
+                    handles.handle, columns=columns, use_threads=bool(use_threads)
+                )
```

Comment:
Do we still need this? (the fix is in a released pyarrow, so with latest of both you won't see any warning. Although of course when testing with an older pyarrow you will see it, but I am not sure we added such warning suppressions for other deprecations like DataFrame(mgr)?)

Reply:
we dont _need_ it, no.  we need to catch this either here or filterwarnings in a bunch of test places.  so i prefer to catch it one place, but OK either way